lib/github/api.nit

```diff
@@ -145,6 +145,19 @@ class GithubAPI
 		return res.as(JsonObject)
 	end
 
+	# Get the Github logged user from `auth` token.
+	#
+	# Loads the `User` from the API or returns `null` if the user cannot be found.
+	#
+	#     var api = new GithubAPI(get_github_oauth)
+	#     var user = api.load_auth_user
+	#     assert user.login == "Morriar"
```

Comment:
nitish? Because `get_github_oauth` will likely not login with `Morriar` on test machines